package/MDAnalysis/core/topologyattrs.py

```diff
@@ -103,14 +114,17 @@ def wrapper(attr, group, values):
 
         if isinstance(group, ComponentBase):
             if not val_len == 0:
-                raise ValueError(_SINGLE_VALUE_ERROR.format(
-                    cls=group.__class__.__name__, attrname=attr.singular,
-                    lenvalues=val_len))
+                raise ValueError(
```

Comment:
I'm just going to drop this one comment here - please don't auto format entire files, it makes blame history very hard to deal with. It's always difficult to make a clear cut with this, please try to apply PEP8 corrections on areas of the file you touch rather than blanket everything.

Reply:
Sorry about that, many of my exchanges in the review of my contributions have been about PEP8 and since the `Contributing to the main codebase <https://userguide.mdanalysis.org/stable/contributing_code.html#working-with-mdanalysis-code>` page recommended yapf I thought I was inline with expectations. Would you like to undo all the automated changes?

Reply:
We really need to make that guide clearer :( Sorry to be asking this, but if it's possible to limit the PEP8 changes to the bits of code you're making new code contributions to that would be great.